src/main/java/org/opentripplanner/middleware/tripmonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -771,6 +764,33 @@ public boolean shouldSkipMonitoredTripCheck(boolean persist) throws Exception {
         return true;
     }
 
+    // Check if previous matching itinerary day is still valid
+    private boolean isPrevMatchingItineraryDayValid() {
+        Calendar calendar = Calendar.getInstance();
+        calendar.setTime(previousMatchingItinerary.startTime);
+
+        int dayOfWeek = calendar.get(Calendar.DAY_OF_WEEK);
```

Comment:
Nit. You could in-line this variable.

Reply:
addressed in [3bd3cb1](https://github.com/ibi-group/otp-middleware/pull/284/commits/3bd3cb1672e02da0b1ed37af5f9e644c42215403)